earth2mip/model_registry.py

```diff
@@ -120,6 +120,30 @@ def metadata(self) -> schema.Model:
             return schema.Model.parse_raw(f.read())
 
 
+# TODO: Replace with concept of NGC model registry
+class FCNPackage(Package):
+    def __init__(self, root: str, seperator: str):
+        super().__init__(root, seperator)
+        self._load_model_package()
+
+    def _load_model_package(self):
+        model_registry = os.path.dirname(self.root)
+        if not os.path.isdir(self.root):
+            logger.info("Downloading FCN model checkpoint, this may take a bit")
+            urllib.request.urlretrieve(
+                "https://api.ngc.nvidia.com/v2/models/nvidia/modulus/modulus_fcn/"
+                + "versions/v0.1/files/fcn.zip",
+                f"{model_registry}/fcn.zip",
+            )
+            # Unzip
+            with zipfile.ZipFile(f"{model_registry}/fcn.zip", "r") as zip_ref:
+                zip_ref.extractall(model_registry)
+            # Clean up zip
+            os.remove(f"{model_registry}/fcn.zip")
+        else:
+            logger.info("FCN package already found, skipping download")
+
```

Comment:
this can be a function:

```suggestion
def  FCNPackage(root, seperator):
        model_registry = os.path.dirname(root)
        if not os.path.isdir(oot):
            logger.info("Downloading FCN model checkpoint, this may take a bit")
            urllib.request.urlretrieve(
                "https://api.ngc.nvidia.com/v2/models/nvidia/modulus/modulus_fcn/"
                + "versions/v0.1/files/fcn.zip",
                f"{model_registry}/fcn.zip",
            )
            # Unzip
            with zipfile.ZipFile(f"{model_registry}/fcn.zip", "r") as zip_ref:
                zip_ref.extractall(model_registry)
            # Clean up zip
            os.remove(f"{model_registry}/fcn.zip")
        else:
            logger.info("FCN package already found, skipping download")
            
        return Package(root, seperator)

```

same thing for DLWPPackage.

Should be simple to generalize to an arbitrary NGC+zip packages....would be nice to do that now to avoid the copy paste.

Reply:
Good call, cleaned this up a bit